ff/serving/coordinator/coordinator.go

```diff
@@ -216,31 +224,28 @@ func (c *Coordinator) runPrimaryTableJob(transformSource *metadata.SourceVariant
 	if _, err := offlineStore.RegisterPrimaryFromSourceTable(providerResourceID, sourceName); err != nil {
 		return err
 	}
-	if err := c.Metadata.SetStatus(context.Background(), resID, string(metadata.READY)); err != nil {
+	if err := c.Metadata.SetStatus(context.Background(), resID, metadata.READY, ""); err != nil {
 		return err
 	}
 	return nil
 }
 
 func (c *Coordinator) runRegisterSourceJob(resID metadata.ResourceID) error {
+	c.Logger.Debug("Running register source job on resource: ", resID)
```

Comment:
Set as info, same as all the other job starts/ends
Essentially all the jobs initiations that start running should be info since they are initiated by either user action or a schedule. If we set this to production mode and log level is info it would be nice to see when jobs start and end